airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -93,15 +93,30 @@ def inner_wrapper(self: "DataflowHook", *args, **kwargs) -> RT:
 class DataflowJobStatus:
     """
     Helper class with Dataflow job statuses.
+    Reference: https://cloud.google.com/dataflow/docs/reference/rest/v1b3/projects.jobs#Job.JobState
```

Comment:
This file is also modified in https://github.com/apache/airflow/pull/8553. I am assuming it is the same changes, skipping this file for the review.

Reply:
I replied here https://github.com/apache/airflow/pull/8553#discussion_r496513620 and changed here https://github.com/apache/airflow/pull/8550/commits/92858ebbc39c242376304d4a9d9106263aed7369